stinit.c

```diff
@@ -333,8 +333,7 @@ find_pars(FILE *dbf, char *company, char *product, char *rev, devdef_tr *defs, i
                 continue;
             }
 
-            strcpy(modebuf, comptr);
-            strcat(modebuf, cp);
+            snprintf(defstr, sizeof(defstr)/sizeof(char), "%s%s", comptr, cp);
```

Comment:
I was trying to debug a weird error only to find this line, which now seems very wrong to me. I can't remember why I didn't see/flag it before… Strange thing to have coverity suggest this.

Anyway, fixing, and thanksful to Travis for letting me know :)